communication/marketing-team/README.md

```diff
@@ -1,57 +1,41 @@
-Future home for all things marketing team for contributor experience
-
-# Charter
-view our charter in this [marketing folder] for more about us and how to join
+The Contributor Marketing Team is part of the community-management subproject in
```

Comment:
It might be worth adding an H1 header here of `Contributor Marketing Team` or something of that sort.

Reply:
fixed